### PR TITLE
Manual 3.10 Part 1 – Event based damage calculator

### PR DESCRIPTION
**Pull request by raoanirudh**
Address the following open items from #3460 

- [x] Document the `event_based_damage` calculator

In addition:
- Update `oq engine --help` text
- Update site association docs
- Add engine release acronyms to glossary
- Update risk output export overview page
- Shorter link to OpenQuake users Google group

### CODE REVIEW

File: doc/adv-manual/asset-site-association.rst [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
-
-It must be noticed that the engine rounds longitudes and latitudes
-to 5 digits (1 meter resolution) so sites that differs at the 6 digit or so
-end up as being duplicated for the engine, and they may give issues.
+3. Otherwise the user can specify a grid via the `region` and
+   `region_grid_spacing` parameters.
+4. Otherwise the sites can be inferred from the exposure, if any,
+   in two different ways:
    
+   4a. if `region_grid_spacing` is specified, a grid is implicitely
```
Comment (micheles) [inline doc/adv-manual/asset-site-association.rst:17]: You removed the part about longitude and latitude rounding, which is quite important.

File: doc/adv-manual/asset-site-association.rst [hunk 2]
```diff
@@ -41,24 +38,17 @@ There are a number of error situations:
...
+----------------------------------------------
 
-Should you have a very large site model file with million of points,
-you can always specify a large enough `region_grid_spacing`: then the
-sites would be inferred from the implicit grid, the site parameters
-would be inferred from the closest site parameters and the assets
-would be associated to the closest grid site. Since the result site
-collection would be small, everything will work efficiently.
+If the user specifies the hazard sites in a CSV file, such sites are
+the ones used to compute the hazard.
```
Comment (micheles) [inline doc/adv-manual/asset-site-association.rst:54]: It is already said in rule 2, isn't it?

File: doc/adv-manual/asset-site-association.rst (hunk 1 above)
Comment (raoanirudh) [reply doc/adv-manual/asset-site-association.rst:17]: Inadvertently, adding it back.

File: doc/adv-manual/asset-site-association.rst (hunk 2 above)
Comment (raoanirudh) [reply doc/adv-manual/asset-site-association.rst:54]: Correct. Removing this line.

**Comment by micheles** [comment]
LGTM and merging. Additional work on the documentation can go into a different PR.

**Pull request merged**